parser/src/main/java/com/linkedin/avroutil1/util/ConfigurableAvroSchemaComparator.java

```diff
@@ -39,30 +39,45 @@
  */
 public class ConfigurableAvroSchemaComparator {
 
+  /**
+   * Compares two AvroSchema objects for equality using the specified SchemaComparisonConfiguration. This method is high
+   * performant and return on the first found mismatch.
```

Comment:
> and return on the first found mismatch.

it returns a boolean so that is false, correct?

Reply:
That is correct, I also have a unit test which tests and confirms this behavior.

Reply:
Ok cool, let's update that comment then :)